Makefile

```diff
@@ -133,9 +136,13 @@ ENVTEST = $(shell pwd)/bin/setup-envtest
 envtest: ## Download envtest-setup locally if necessary.
 	$(call go-get-tool,$(ENVTEST),sigs.k8s.io/controller-runtime/tools/setup-envtest@latest)
 
-# go-get-tool will 'go install' any package $2 and install it to $1.
 PROJECT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
-define go-get-tool
+GOMOCK = $(shell pwd)/bin/mockgen
+gomock:
+	$(call go-get-tool,$(GOMOCK),github.com/golang/mock/mockgen@v1.6.0)
+
+# go-install-tool will 'go install' any package $2 and install it to $1.
+define go-install-tool
```

Comment:
Please, revert this to `go-get-tool`. There is no reference to `go-install-tool` in this file, for release-4.12

Reply:
ok I've removed the commit and I ran `go mod tidy && go mod vendor`. I hope this is fine.